hail/python/test/hail/methods/test_qc.py

```diff
@@ -463,3 +469,32 @@ def tuple2(arr):
         dataproc_result = parse_lof_info_into_dict(dataproc_result)
 
         assert hail_vep_result._same(dataproc_result)
+
+
+    @qobtest
+    @skip_unless_service_backend(clouds=['gcp'])
+    @set_gcs_requester_pays_configuration(GCS_REQUESTER_PAYS_PROJECT)
+    @test_timeout(batch=10 * 60)
+    def test_vep_grch38_using_indexed_cache(self):
```

Comment:
While this describes the motivation, this test is really just testing that we can run VEP on loci with large positions. I prefer that we describe what the test is rather than what it aspires to be.